CMakeLists.txt

```diff
@@ -309,19 +323,29 @@ if (ADDRESS_SANITIZER_DEBUG)
   )
 endif()
 
+
+# Note: we need to provide a support for Eigen backend
+#       for both gencode and SeisSol
+target_include_directories(SeisSol-gencode PUBLIC
```

Comment:
Hm, why is this PUBLIC? Shouldn't this be private?

Reply:
So, as the comment says we need eigen (and yateto) headers for the code generated by `yateto` (in case of Eigen backend) and `SeisSol-lib`. If it is `PRIVATE` then the includes will be applied only for `SeisSol-gencode` target.

Reply:
Yes, we are using. I mean the path to eigen will be applied to `SeisSol-gencode` and everything which depends on `SeisSol-gencode`. In our case, it is `SeisSol-lib` and so on (SeisSol-bin, SeisSol-proxy, tests, etc.).